synthesizer/src/vm/finalize.rs

```diff
@@ -995,8 +999,13 @@ impl<N: Network, C: ConsensusStorage<N>> VM<N, C> {
                     bond_validator_addresses.into_iter().filter(|address| !committee_members.contains(address)).count();
                 // Compute the next committee size.
                 let next_committee_size = committee_members.len().saturating_add(num_new_validators);
+                // Determine the maximum committee size to use.
+                let max_committee_size = match state.block_height() < N::CONSENSUS_V3_HEIGHT {
```

Comment:
Given that we know there will be future additions to this: we can match just on the block height, and compare in the branches `if height < v3_height`. Then in the future we only have to adjust two branches.

Ideally that same format is used for all consensus height matches, but I usually defer formatting refactors until necessary.